avsomr/avsimport.cpp

```diff
@@ -81,10 +83,15 @@ static Score::FileError doImportMSMR(MasterScore* score
             if (err != Score::FileError::FILE_NO_ERROR)
                   return err;
 
+            // corrections after import
             Text* title = score->getText(Tid::TITLE);
             if (title)
                   title->setPlainText(baseName);
 
+            score->setMetaTag("movementNumber", "");
+            score->setMetaTag("movementTitle", "");
+            score->setMetaTag("source", QFileInfo(filePath).fileName());
+            score->setMetaTag("workTitle", baseName);
```

Comment:
There's no Title Frame and -Text to be used instead yet, is it?

Reply:
Oh, I see, it does get set further up, and to same same value...

Reply:
Hmm, no... better use `score->setMetaTag("workTitle", title);` here

Reply:
changed code, but But in fact, at the moment, I have no information about the title of the score. Audiveris writes in the title "[Audiveris detected movement]", so I decided it was better to write the baseName of the file in the title, which I do a few lines above.